### PR TITLE
(DOCSP-40177): Replace mongodump sharded cluster behavior with link t…

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
…o tutorial

## DESCRIPTION

Replace standalone mongodump behavior details with link to the recently-updated tutorial.

## STAGING

https://preview-mongodbjeffallenmongo.gatsbyjs.io/database-tools/DOCSP-40177/mongodump/mongodump-behavior/#sharded-clusters

## JIRA

https://jira.mongodb.org/browse/DOCSP-40177

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=665f809fcf5bec1c2965e4af

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by jmd-mongo — APPROVED** [review]
Thanks for this fix, @jeff-allen-mongo! This one LGTM.

Thanks,
Joe

File: source/includes/extracts-sharded-clusters-backup-restore-limitations.yaml [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
-   of transactions across shards.
+   To use :program:`mongodump` and :program:`mongorestore` as a backup
+   strategy for sharded clusters, see :ref:`backup-sharded-dumps`.
 
-   For 4.2+ sharded clusters with in-progress sharded transactions, use
-   one of the following coordinated backup and restore processes which
-   *do maintain* the atomicity guarantees of transactions across shards:
+   Sharded clusters can also use one of the following coordinated backup
+   and restore processes, which maintain the atomicity guarantees of
+   transactions across shards:
```
Comment (mvankeulen94) [inline source/includes/extracts-sharded-clusters-backup-restore-limitations.yaml:9]: Would it make sense to also mention that these solutions provide these atomicity guarantees while still accepting writes?

**Review by mvankeulen94 — COMMENTED** [review]
Thank you for working on this! Added a comment for consideration.

File: source/includes/extracts-sharded-clusters-backup-restore-limitations.yaml (hunk 1 above)
Comment (jeff-allen-mongo) [reply source/includes/extracts-sharded-clusters-backup-restore-limitations.yaml]: Updated!

**Comment by jeff-allen-mongo** [comment]
@mvankeulen94 thanks for the feedback! I tweaked the copy per your suggestion. Could you take another look?

**Review by mvankeulen94 — APPROVED** [review]
sounds great, thank you!

**Pull request merged**